packages/coin98-wallet/src/lib/coin98-wallet.ts

```diff
@@ -0,0 +1,206 @@
+    }
+
+    return transactions.map((transaction) => {
+      return {
+        signerId: transaction.signerId || account.accountId,
+        receiverId: transaction.receiverId || contract.contractId,
+        actions: transaction.actions,
+      };
+    });
+  };
+
+  return {
+    async signIn({ contractId }) {
+      const existingAccounts = getAccounts();
+
+      if (existingAccounts.length) {
+        return existingAccounts;
+      }
+
+      await _state.wallet.near.connect({ prefix: "near_selector", contractId });
+      return getAccounts();
+    },
+
+    async signOut() {
+      // Ignore if unsuccessful (returns false).
+      await _state.wallet.near.disconnect();
+    },
+
+    async getAccounts() {
+      return getAccounts();
+    },
+
+    async verifyOwner({ message }) {
+      const account = getActiveAccount(store.getState());
+
+      if (!account) {
+        throw new Error("No active account");
+      }
+
+      const accountId = account.accountId;
+      const pubKey = await _state.wallet.near.signer.getPublicKey(accountId);
+      const block = await provider.block({ finality: "final" });
+
+      const data = {
+        accountId,
+        message,
+        blockId: block.header.hash,
+        publicKey: Buffer.from(pubKey.data).toString("base64"),
+        keyType: pubKey.keyType,
+      };
+      const encoded = JSON.stringify(data);
+
+      throw new Error(`Method not supported by ${metadata.name}`);
+
+      const signed = await _state.wallet.near.signer.signMessage(
+        new Uint8Array(Buffer.from(encoded)),
+        accountId,
+        options.network.networkId
+      );
+
+      return {
+        ...data,
+        signature: Buffer.from(signed.signature).toString("base64"),
+        keyType: signed.publicKey.keyType,
+      };
+    },
+
+    async signAndSendTransaction({ signerId, receiverId, actions }) {
+      logger.log("signAndSendTransaction", { signerId, receiverId, actions });
+      const signedTransactions = await signTransactions(
+        transformTransactions([{ signerId, receiverId, actions }]),
+        _state.wallet.near.signer,
+        options.network
+      );
+
+      return provider.sendTransaction(signedTransactions[0]);
+    },
+
+    async signAndSendTransactions({ transactions }) {
+      logger.log("signAndSendTransactions", { transactions });
+
+      const signedTransactions = await signTransactions(
+        transformTransactions(transactions),
+        _state.wallet.near.signer,
+        options.network
+      );
+
+      logger.log(
+        "signAndSendTransactions:signedTransactions",
+        signedTransactions
+      );
+
+      const results: Array<FinalExecutionOutcome> = [];
+
+      for (let i = 0; i < signedTransactions.length; i++) {
+        results.push(await provider.sendTransaction(signedTransactions[i]));
+      }
+
+      return results;
+    },
+  };
+};
+
+export const setupCoin98Wallet = ({
+  iconUrl = defaultCoin98Logo
```

Comment:
Can you change this so it is like the other packages. Using a string path and assets folder.